lib/timeline/timeline.dart

```diff
@@ -60,13 +80,17 @@ class TimelineScreen extends Screen {
       ..disabled = true
       ..click(_resumeRecording);
 
-    final PButton trackWidgetBuildsButton = new PButton('Track widget builds')
-      ..small();
-    final PButton perfOverlayButton = new PButton('Performance overlay')
-      ..small();
-    final PButton repaintRainbowButton = new PButton('Repaint rainbow')
-      ..small();
-    final PButton debugDrawButton = new PButton('Debug draw')..small();
+    PButton createButton(String text, bool shouldDisableForProfileMode) {
+      return new PButton(text)
+        ..small()
+        ..disabled = shouldDisableForProfileMode && isProfileMode
+        ..tooltip = isProfileMode ? 'Unavailable on a profile build.' : '';
```

Comment:
better is to create the UI right away with the button disabled and then enable it async when we verify the app is debug mode.